miri/datamodels/ancestry.py

```diff
@@ -45,6 +47,7 @@
                'MiriBadPixelMaskModel' :    'MaskModel', \
                'MiriDarkReferenceModel' : 'DarkMIRIModel', \
                'MiriImagingDistortionModel' : 'DistortionModel', \
+               'MiriLrsD2WModel' : 'SpecwcsModel', \
```

Comment:
For future reference, this maps to the SpecwcsModel at:
https://github.com/spacetelescope/jwst/blob/74ed6b27b0537135c2dab1a989f412121f19c3e2/jwst/datamodels/wcs_ref_models.py#L157